src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -291,6 +297,14 @@ def load_command_table(self, _):
         g.custom_command('show-log', 'get_deployment_script_logs')
         g.custom_command('delete', 'delete_deployment_script', confirmation=True)
 
+    with self.command_group('ts', resource_templatespecs_sdk, resource_type=PROFILE_TYPE, is_preview=True, min_api='2019-06-01-preview') as g:
+        g.custom_command('create', 'create_template_spec', validator=_validate_template_input)
+        g.custom_command('update', 'update_template_spec', validator=_validate_template_input, confirmation=True)
```

Comment:
Two questions here:
1. why adding confirmation for update?
2. is update command PATCH like operation? If not, please use generic_update_command

Reply:
Update is a patch like operation